tests/test_utils.py

```diff
@@ -20,3 +23,66 @@ def test_load_custom_schema(tmpdir):
     temp = tmpdir.join("custom_schema.json")
     temp.write('{"foo": "bar"}')
     assert pyhf.utils.load_schema(temp.strpath)
+
+
+def test_hypotest(tmpdir):
```

Comment:
@kratsg Suggestions for writing a more efficient testing scheme are welcome.

Reply:
This looks good enough to me for now. Perhaps splitting up the test into multiple test functions make more sense.. e.g. `test_hypotest_return_p_values` and so on.